packages/dialtone-tokens/tokens/semantic/tmo/dark.json

```diff
@@ -5,6 +5,14 @@
         "value": "#F285BC",
         "type": "color"
       }
+    },
+    "mention": {
+      "color": {
+        "background-strong": {
+          "value": "#FFCEE7",
+          "type": "color"
+        }
+      }
     }
```

Comment:
TMO will look like this...

![image](https://github.com/user-attachments/assets/f22c5db2-81a0-4077-847d-59bcec0cccf9)